### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by JakobMiksch**
- adds basic tests for utils
- also adds some docs in the production code and does renaming

### CODE REVIEW

**Comment by JakobMiksch** [comment]
cc @jansule

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/mapUtil.test.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Map from 'ol/Map.js';
+import OSM from 'ol/source/OSM.js';
+import TileLayer from 'ol/layer/Tile.js';
+import View from 'ol/View.js';
+import { ChartConfig } from 'plugins/plugin-chart-cartodiagram/src/types';
```
Comment (JakobMiksch) [inline superset-frontend/plugins/plugin-chart-cartodiagram/test/util/mapUtil.test.ts:24]: make relative import

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/serviceUtil.test.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { isVersionBelow } from '../../src/util/serviceUtil';
+
+describe('isVersionBelow', () => {
+  it('returns correct results', () => {
```
Comment (JakobMiksch) [inline superset-frontend/plugins/plugin-chart-cartodiagram/test/util/serviceUtil.test.ts:23]: make 4 describes

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/mapUtil.test.ts (hunk 1 above)
Comment (JakobMiksch) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/mapUtil.test.ts]: done

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/serviceUtil.test.ts (hunk 2 above)
Comment (JakobMiksch) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/serviceUtil.test.ts]: done

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/chartUtil.test.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  it('returns pie chart', () => {
+    const result = createChartComponent('pie', config, width, height);
+    const createdType = result.type.name;
+
+    expect(createdType).toEqual('EchartsPie');
+  });
+
+  it('returns classic timeseries', () => {
+    const timeSeriesTypes = [
```
[the remark this answers is not in the document]
Comment (JakobMiksch) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/chartUtil.test.ts]: done e796b0b17

File: superset-frontend/plugins/plugin-chart-cartodiagram/test/util/controlPanelUtil.test.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    const value = undefined;
+
+    const result = selectedChartMutator(response, value);
+    expect(result[0].label).toEqual(sliceName1);
+    expect(result[1].label).toEqual(sliceName2);
+  });
+
+  it('returns correct result if both value and response are provided', () => {
+    // TODO
+    expect(selectedChartMutator).toBeDefined();
```
[the remark this answers is not in the document]
Comment (JakobMiksch) [reply superset-frontend/plugins/plugin-chart-cartodiagram/test/util/controlPanelUtil.test.tsx]: good point!!
done with cfbad454a

**Comment by JakobMiksch** [comment]
@jansule thanks for the review. I applied your remarks.

**Review by jansule — APPROVED** [review]
Thanks for addressing the remarks!

**Pull request merged**